charts/hedera-mirror-importer/templates/statefulset.yaml

```
@@ -86,6 +89,9 @@ spec:
         - name: data
           emptyDir: {}
 {{- else }}
+{{- if .Values.extraVolumes }}
+{{ toYaml .Values.extraVolumes | nindent 8}}
+{{- end }}
```

```suggestion
{{- if .Values.extraVolumes }}
{{- toYaml .Values.extraVolumes | nindent 8 }}
{{- end }}
```

also this block is placed at the wrong place, should be before:

```yaml
{{- if not .Values.persistence.enabled }}
```

---

Good catch. Those collapsed diffs make it hard to see.

---

Updated